src/debug_clang.cc

```diff
@@ -463,6 +463,7 @@ void Debug::Clang::remove_breakpoint(const boost::filesystem::path &file_path, i
             auto file_spec=line_entry.GetFileSpec();
             boost::filesystem::path breakpoint_path=file_spec.GetDirectory();
             breakpoint_path/=file_spec.GetFilename();
+            breakpoint_path = boost::filesystem::canonical(breakpoint_path);
```

Comment:
lldb is not 100% stable, at least not the earlier versions, it's just to be on the safe side. No need in my opinion to handle the error in this case, as it probably never will happen.